src/pages/index.js

```diff
@@ -93,34 +254,62 @@ const cardSection = new Section(
   },
   cardClassSelector
 );
+*/
 
 const imagePopup = new PopupWithImage({ popupSelector: previewModalSelector });
 
 const addDestinationPopup = new PopupWithForm({
   popupSelector: addDestinationSelector,
   handleFormSubmit: (formData) => {
     console.log(formData);
-
+    console.log(formData);
     formData.link = formData.destination_image_URL;
     delete formData.destination_image_URL;
 
     formData.name = formData.destination_title;
     delete formData.destination_title;
+    api
+      .addNewCard(formData)
+      .then(() => {
+        //if (res.ok) {
 
-    cardSection.addItem(formData); //BULLET POINT #12 RESOLUTION
-    //test
+        return api
+          .getCards()
+          .then((cards) => {
+            console.log(cards[0]._id);
+            return cards[0]._id;
+          })
+          .then((res) => {
+            formData._id = res;
+            console.log(
+              "#\\3" +
+                formData._id.slice(1, 2) +
+                " " +
+                formData._id.slice(2, formData._id.length)
+            );
+            cardSection.addItem(formData);
+            document.querySelector(
+              "#\\3" +
+                formData._id.slice(1, 2) +
+                " " +
+                formData._id.slice(2, formData._id.length)
```

Comment:
Please, don't do that here. If you want you can do it inside `Card`. It's responsibility of that class to handle anything related to cards


And you don't need to search cards here. You have the card inside `Card`